frontend/src/lib/introductions/GroupsIntroductionOption.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { useValues } from 'kea'
+import { LockOutlined } from '@ant-design/icons'
+import Select from 'rc-select'
+import { Link } from 'lib/components/Link'
+import { groupsAccessLogic, GroupsAccessStatus } from 'lib/introductions/groupsAccessLogic'
+import { IconOffline } from 'lib/components/icons'
+
+export function GroupsIntroductionOption({ value }: { value: any }): JSX.Element | null {
+    const { groupsAccessStatus, upgradeLink } = useValues(groupsAccessLogic)
+
+    if (
+        ![GroupsAccessStatus.HasAccess, GroupsAccessStatus.HasGroupTypes, GroupsAccessStatus.NoAccess].includes(
+            groupsAccessStatus
+        )
+    ) {
+        return null
+    }
+
+    return (
+        <Select.Option
+            key="groups"
+            value={value}
+            disabled
+            style={{
+                height: '100%',
+                width: '100%',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                backgroundColor: 'var(--bg-side)',
+                color: 'var(--text-muted)',
+            }}
+        >
+            <div
+                style={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'space-between',
+                }}
+            >
+                <span>
+                    <LockOutlined style={{ marginRight: 4, color: 'var(--warning)' }} />
+                    Unique groups
+                    <Link
+                        to="https://posthog.com/docs/user-guides/group-analytics?utm_medium=in-product&utm_campaign=group-analytics-learn-more"
```

Comment:
```suggestion
                        to="https://posthog.com/docs/user-guides/group-analytics#availability?utm_medium=in-product&utm_campaign=group-analytics-learn-more"
```

And suggest creating an availability section in the guide that mentions the plan/license required to have Group analytics (with relevant links).